libautoscoper/src/gpu/cuda/RayCaster.cpp

```diff
@@ -61,6 +61,7 @@ RayCaster::RayCaster() : volumeDescription_(0),
     std::stringstream name_stream;
     name_stream << "DrrRenderer" << (++num_ray_casters);
     name_ = name_stream.str();
+    visible_ = true;
```

Comment:
Consider moving this commit its own PR and reference the commit it is following up on.

Reply:
See #251